### PR TITLE
Support textStyle

### LINKED ISSUE #40: Add textStyle support
Hello,

I'd like to have textStyle support too so I can make my text bold. I am using the textAppearance tag but it does not support textStyle="bold" yet.

Thanks!

### PR DESCRIPTION
**Pull request by jinatonic**
closes #40

### CODE REVIEW

File: ticker/src/main/java/com/robinhood/ticker/TickerView.java [hunk 1]
```diff
@@ -149,6 +150,23 @@ protected void init(Context context, AttributeSet attrs, int defStyleAttr, int d
             textPaint.setShadowLayer(styledAttributes.shadowRadius, styledAttributes.shadowDx,
                     styledAttributes.shadowDy, styledAttributes.shadowColor);
         }
+        if (styledAttributes.textStyle != 0) {
+            final int textStyle = styledAttributes.textStyle;
+            int flags = textPaint.getFlags();
+            if ((textStyle & AssistStructure.ViewNode.TEXT_STYLE_BOLD) != 0) {
```
Comment (danh32) [inline ticker/src/main/java/com/robinhood/ticker/TickerView.java]: seems really weird that this is down in the accessibility package

File: ticker/src/main/java/com/robinhood/ticker/TickerView.java [hunk 2]
```diff
@@ -149,6 +150,23 @@ protected void init(Context context, AttributeSet attrs, int defStyleAttr, int d
...
+                flags |= Paint.FAKE_BOLD_TEXT_FLAG;
+            }
+            if ((textStyle & AssistStructure.ViewNode.TEXT_STYLE_ITALIC) != 0) {
+                textPaint.setTextSkewX(-0.25f);
```
Comment (danh32) [inline ticker/src/main/java/com/robinhood/ticker/TickerView.java]: magic number. how does TextView do this?

Comment (jinatonic) [inline ticker/src/main/java/com/robinhood/ticker/TickerView.java]: good point, TextView sets a flag on the typeface object so changed approach to that way.

File: ticker/src/main/java/com/robinhood/ticker/TickerView.java (hunk 1 above)
Comment (jinatonic) [inline ticker/src/main/java/com/robinhood/ticker/TickerView.java]: yeah I was looking at the wrong textview source code, should use constants from `Typeface`.

**Pull request merged**